### ISSUE TITLE
Page reloads every 5 seconds, part 2 (Patch included!)

### ISSUE THREAD

**Issue by alxwr** [nextcloud/polls#344]
Exactly same problem as in #283 

Using OC 10.0.08 with polls 0.8.1.

This patch fixes the issue:

```diff
--- apps/polls/js/vote.js.orig  2018-06-07 20:13:40.915775000 +0200
+++ apps/polls/js/vote.js       2018-06-07 20:14:29.568190000 +0200
@@ -48,7 +48,7 @@
 
 function updateAvatar(obj) {
        // Temporary hack - Check if we have Nextcloud or ownCloud with an anomymous user
-       if (!document.getElementById('nextcloud') && OC.currentUser === '') {
+       if (!document.getElementById('nextcloud') && (OC.currentUser === '' || OC.currentUser === false)) {
                $(obj).imageplaceholder(obj.title);
        } else {
                $(obj).avatar(obj.title, 32);
```

**Comment by joergmschulz** [comment]
cannot reproduce it in the beta2 for 9.0 / can you confirm and mark this as closed?

**Comment by AdminValida** [comment]
We applied the patch but still do have the same problem with ownCloud 10.0.9 and polls 0.8.1

Regards

**Comment by dartcafe** [comment]
I try to reproduce it again and aim the fix to version 0.8.2
